tests/FSharp.Compiler.Service.Tests/FSharp.Compiler.Service.Tests.fsproj

```diff
@@ -116,6 +116,9 @@
     <Compile Include="..\service\SyntaxTreeTests\SynIdentTests.fs">
       <Link>SyntaxTree\SynIdentTests.fs</Link>
     </Compile>
+    <Compile Include="..\service\SyntaxTreeTests\AttributeTests.fs">
+      <Link>SyntaxTree\AttributeTests.fs</Link>
```

Comment:
Could you also move other attribute related parser tests to this file? Would be great to have them in one place

Reply:
The other tests that have attributes are more related to where they are used.
For example, ensure an attribute on a type is included in the type range. 
I think it makes more sense to keep that test in the `TypeTests` file.

Reply:
I'm sorry, I should've added a link to the tests to the original message. 🙂
I was thinking about this test (I thought there were more): https://github.com/dotnet/fsharp/blob/4f7f156c96e37e2bc211ec9b9b109bbbfaa688b4/tests/service/ServiceUntypedParseTests.fs#L106